02_configure_host.sh

```diff
@@ -33,7 +33,7 @@ if [ ! -z "${VM_NODES_FILE}" ]; then
   exit 1
 fi
 
-if [[ "${USE_IPV4}" != "true" ]]; then
+if [[ "${IP_STACK}" != "v6" ]]; then
```

Comment:
Should be `==` or `!= "v4"` no?

Reply:
yes, good catch.  thank you.  i pushed before testing